ui/packages/consul-ui/app/templates/dc/services/show/topology.hbs

```diff
@@ -27,66 +27,107 @@ as |route|>
-    <CollapsibleNotices @collapsible={{collapsible}}>
-      {{#if topology.FilteredByACLs}}
-        <TopologyMetrics::Notice
-          data-test-notice='filtered-by-acls'
-          @type="info"
-          @for="limited-access"
-          @action={{false}}
-        />
-      {{/if}}
-      {{#if (eq dc.DefaultACLPolicy 'allow')}}
-        <TopologyMetrics::Notice
-          data-test-notice='default-allow'
-          @type="warning"
-          @for="default-allow"
-          @internal={{true}}
-          @action={{true}}
-        />
-      {{/if}}
-      {{#if topology.wildcardIntention}}
-        <TopologyMetrics::Notice
-          data-test-notice='wildcard-intention'
-          @type="warning"
-          @for="wildcard-intention"
-          @internal={{true}}
-          @action={{true}}
-        />
-      {{/if}}
-      {{#if topology.notDefinedIntention}}
-        <TopologyMetrics::Notice
-          data-test-notice='not-defined-intention'
-          @type="warning"
-          @for="not-defined-intention"
-          @link="{{env 'CONSUL_DOCS_URL'}}/connect/registration/service-registration#upstreams"
-          @internal={{false}}
-          @action={{true}}
-        />
-      {{/if}}
-      {{#if (and topology.noDependencies (can 'use acls'))}}
-        <TopologyMetrics::Notice
-          data-test-notice='no-dependencies'
-          @type="info"
-          @for="no-dependencies"
-          @link="{{env 'CONSUL_DOCS_URL'}}/connect/registration/service-registration#upstream-configuration-reference"
-          @internal={{false}}
-          @action={{true}}
-        />
-      {{/if}}
-      {{#if (and topology.noDependencies (not (can 'use acls')))}}
-        <TopologyMetrics::Notice
-          data-test-notice='acls-disabled'
-          @type="warning"
-          @for="acls-disabled"
-          @link="{{env 'CONSUL_DOCS_URL'}}/security/acl/acl-system#configuring-acls"
-          @internal={{false}}
-          @action={{true}}
-        />
-      {{/if}}
-    </CollapsibleNotices>
-  {{/let}}
+
+    <div
+      class="topology-notices"
+    >
+      <Disclosure
+        @expanded={{true}}
+        as |disclosure|
+      >
+        {{! Make a map of key=enabled to tell us which notices are enabled }}
+        {{! The `false` here is so we can easily open all the notices up by changing to `true` }}
+        {{#let
+          (from-entries (array
+            (array 'filtered-by-acls' (or false topology.FilteredByACLs))
+            (array 'default-allow' (or false (eq dc.DefaultACLPolicy 'allow')))
+            (array 'wildcard-intention' (or false topology.wildcardIntention))
+            (array 'not-defined-intention' (or false topology.notDefinedIntention))
+            (array 'no-dependencies' (or false (and topology.noDependencies (can 'use acls'))))
+            (array 'acls-disabled' (or false (and topology.noDependencies (not (can 'use acls')))))
+          ))
```

Comment:
If you change all of the `false`s here in the `or`s to `true` it'll show you all panels and therefore the disclosure-ness kicks in (if you can see less than 3 panels you don't get disclosure-ness)